Source/WebCore/rendering/svg/SVGRenderSupport.cpp

```diff
@@ -207,17 +207,13 @@ static inline void invalidateResourcesOfChildren(RenderElement& renderer)
 static inline bool layoutSizeOfNearestViewportChanged(const RenderElement& renderer)
 {
     const RenderElement* start = &renderer;
-    while (start && !start->isSVGRootOrLegacySVGRoot() && !is<RenderSVGViewportContainer>(*start))
+    while (start && !is<LegacyRenderSVGRoot>(*start) && !is<LegacyRenderSVGViewportContainer>(*start))
```

Comment:
Why the first part of this change?

Reply:
Ah, SVGREnderSupport is not used at all for LBSE -- thus handling RenderSVGRoot is not necessary.